pkg/cmd/run/shared/shared.go

```diff
@@ -213,6 +231,7 @@ func getRuns(client *api.Client, repo ghrepo.Interface, path string, limit int)
 		query := parsed.Query()
 		query.Set("per_page", fmt.Sprintf("%d", perPage))
 		query.Set("page", fmt.Sprintf("%d", page))
+		opts.updateQuery(&query)
```

Comment:
I think I would rather see the `getRuns` function be responsible for constructing the query instead of offloading the work to the `FilterOptions`. 
```golang
if opts != nil {
	if opts.Branch != "" {
 		query.Set("branch", opts.Branch)
 	}
 	if opts.Actor != "" {
 		query.Set("actor", opts.Actor)
 	}
}
```
Additionally adding the `nil` check here would allow us to remove the `EmptyFilters()` function and just pass in `nil` when we don't want to filter by branch or actor.

Reply:
Sounds good to me, I'll update the PR